index.js

```diff
@@ -2,19 +2,11 @@
 const path = require('path');
 
 module.exports = (childPath, parentPath) => {
-	childPath = path.resolve(childPath);
-	parentPath = path.resolve(parentPath);
-
-	if (process.platform === 'win32') {
-		childPath = childPath.toLowerCase();
-		parentPath = parentPath.toLowerCase();
-	}
-
-	if (childPath === parentPath) {
-		return false;
-	}
-
-	parentPath += parentPath.endsWith(path.sep) ? '' : path.sep;
-
-	return childPath.startsWith(parentPath);
+	const relation = path.relative(parentPath, childPath);
+	return Boolean(
+		relation &&
+		relation !== '..' &&
+		!relation.startsWith('..' + path.sep) &&
```

Comment:
```suggestion
		!relation.startsWith(`..${path.sep}`) &&
```

Reply:
Done. 👍 

Would be nice to codify that in XO. I find `${}` somewhat less readable than <code>&nbsp;+&nbsp;</code>, so I have a habit of only using template strings if it actually shortens the code, which it does in some cases, but not here.